java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -803,6 +803,16 @@ public final ColumnVector mergeAndSetValidity(BinaryOp mergeOp, ColumnView... co
     return new ColumnVector(bitwiseMergeAndSetValidity(getNativeView(), columnViews, mergeOp.nativeId));
   }
 
+  /**
+   * Creates a deep copy of a column while replacing the null mask. The null mask is the
```

Comment:
The docs should cover the semantics if the new validity mask makes a row non-null that was null before.

Reply:
Ah, sorry I missed replying to this. (I hadn't realized that you'd caught this.)

I have documented that a formerly invalid row would have an undefined if marked valid, just as in `libcudf`.